### ISSUE TITLE
Code for evaluation with GPT-3.5?

### ISSUE THREAD

**Issue by RuskinManku** [THUDM/LongBench#69]
The results mention the scores of GPT-3.5 but I don't see how I can evaluate GPT using the code as it doesn't have that model.

**Comment by bys0318** [comment]
The GPT-3.5-Turbo-16k model evaluated in our paper has already been deprecated. You can try gpt-3.5-turbo-0125 (16k), or the most recent gpt-4o-mini (128k), according to OpenAI (https://platform.openai.com/docs/models).

**Comment by RuskinManku** [comment]
Thanks for responding. Yes I can evaluate those, but I didn't find code where I can just change the open ai model and evaluate different ones.

**Comment by bys0318** [comment]
Right. We didn't provide code for evaluating API models. You can modify the [pred()](https://github.com/THUDM/LongBench/blob/main/pred.py#L51) fucntion to do so.
